### PR TITLE
(PLATFORM-3929) Enable migration notice on all remaining wikis

### PR DESCRIPTION
**Pull request by Grunny**
Enable migration notice on all remaining wikis except those that may migrate to
wikia.org which will be set in WikiFactory.

/cc @Wikia/core-platform-team

### CODE REVIEW

File: includes/wikia/VariablesBase.php [hunk 1]
```diff
@@ -8889,6 +8889,13 @@
  */
 $wgFandomComMigrationScheduled = false;
 
+/**
+ * Whether the community is scheduled to be migrated to a fandom.com domain, triggers a banner notification
+ * @see SEO-669
+ * @var bool $wgFandomComMigrationScheduled
```
Comment (themech) [inline includes/wikia/VariablesBase.php]: I can see some copy-pasting here :)

**Review by themech — APPROVED** [review]

File: includes/wikia/VariablesBase.php [hunk 2]
```diff
@@ -8889,6 +8889,13 @@
  */
 $wgFandomComMigrationScheduled = false;
 
+/**
+ * Whether the community is scheduled to be migrated to a fandom.com domain, triggers a banner notification
```
Comment (themech) [inline includes/wikia/VariablesBase.php]:
```suggestion
 * Whether the community is scheduled to be migrated to a wikia.org domain
```

File: includes/wikia/VariablesBase.php (hunk 1 above)
Comment (themech) [inline includes/wikia/VariablesBase.php]:
```suggestion
 * @var bool $wgWikiaOrgMigrationScheduled
```

File: includes/wikia/VariablesBase.php [hunk 3]
```diff
@@ -8889,6 +8889,13 @@
...
+ * @see SEO-669
```
Comment (themech) [inline includes/wikia/VariablesBase.php]:
```suggestion
```

File: includes/wikia/VariablesBase.php (hunk 1 above)
Comment (Grunny) [reply includes/wikia/VariablesBase.php]: :blush:

**Review by themech — APPROVED** [review]

**Pull request merged**